### ISSUE TITLE
[AE rv5]Game crashed when Initializing game

### ISSUE THREAD

**Issue by Flithor** [AppliedEnergistics/Applied-Energistics-2#3124]
AE2 rv5-preview

> ---- Minecraft Crash Report ----
> 
> WARNING: coremods are present:
>   JechCore (JustEnoughCharacters-1.12.0-2.0.1.jar)
>   Inventory Tweaks Coremod (InventoryTweaks-1.63+dev.98.jar)
>   AppleCore (AppleCore-mc1.12-2.2.2.jar)
> Contact their authors BEFORE contacting forge
> 
> // Don't do that.
> 
> Time: 10/1/17 1:34 PM
> Description: Initializing game
> 
> **_java.lang.NoSuchFieldError: COAL_**
> 	**_[at appeng.core.features.registries.grinder.GrinderRecipeManager\<init\>(GrinderRecipeManager.java:69)](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/2ed7a5598a61ba84ef98e2eca5d466909c0a65fc/src/main/java/appeng/core/features/registries/grinder/GrinderRecipeManager.java#L69)_**
> 	at appeng.core.features.registries.RegistryContainer.\<init\>(RegistryContainer.java:52)
> 	at appeng.core.Api.\<init\>(Api.java:53)
> 	at appeng.core.Api.\<clinit\>(Api.java:41)
> 	at appeng.core.Registration.registerBlocks(Registration.java:315)
> 	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_10_Registration_registerBlocks_Register.invoke(.dynamic)
> 	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90)
> 	at net.minecraftforge.fml.common.eventhandler.EventBus$1.invoke(EventBus.java:143)
> 	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:179)
> 	at net.minecraftforge.registries.GameData.fireRegistryEvents(GameData.java:721)
> 	at net.minecraftforge.fml.common.Loader.preinitializeMods(Loader.java:605)
> 	at net.minecraftforge.fml.client.FMLClientHandler.beginMinecraftLoading(FMLClientHandler.java:266)
> 	at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:466)
> 	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:377)
> 	at net.minecraft.client.main.Main.main(SourceFile:123)
> 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
> 	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
> 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
> 	at java.lang.reflect.Method.invoke(Unknown Source)
> 	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
> 	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
> 
> 
> A detailed walkthrough of the error, its code path and all known details is as follows:
> ---------------------------------------------------------------------------------------
> 
> -- Head --
> Thread: Client thread
> Stacktrace:
> 	at appeng.core.features.registries.grinder.GrinderRecipeManager.\<init\>(GrinderRecipeManager.java:69)
> 	at appeng.core.features.registries.RegistryContainer.\<init\>(RegistryContainer.java:52)
> 	at appeng.core.Api.\<init\>(Api.java:53)
> 	at appeng.core.Api.\<clinit\>(Api.java:41)
> 	at appeng.core.Registration.registerBlocks(Registration.java:315)
> 	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_10_Registration_registerBlocks_Register.invoke(.dynamic)
> 	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90)
> 	at net.minecraftforge.fml.common.eventhandler.EventBus$1.invoke(EventBus.java:143)
> 	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:179)
> 	at net.minecraftforge.registries.GameData.fireRegistryEvents(GameData.java:721)
> 	at net.minecraftforge.fml.common.Loader.preinitializeMods(Loader.java:605)
> 	at net.minecraftforge.fml.client.FMLClientHandler.beginMinecraftLoading(FMLClientHandler.java:266)
> 	at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:466)
> 
> -- Initialization --
> Details:
> Stacktrace:
> 	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:377)
> 	at net.minecraft.client.main.Main.main(SourceFile:123)
> 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
> 	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
> 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
> 	at java.lang.reflect.Method.invoke(Unknown Source)
> 	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
> 	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
> 
> -- System Details --
> Details:
> 	Minecraft Version: 1.12
> 	Operating System: Windows 10 (amd64) version 10.0
> 	Java Version: 1.8.0_144, Oracle Corporation
> 	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
> 	Memory: 211957880 bytes (202 MB) / 523571200 bytes (499 MB) up to 3110862848 bytes (2966 MB)
> 	JVM Flags: 2 total; -Xincgc -Xmx3000M
> 	IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
> 	FML: MCP 9.40 Powered by Forge 14.21.1.2443 Optifine OptiFine_1.12_HD_U_C4 33 mods loaded, 33 mods active
> 	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
> 	UCH	minecraft{1.12} [Minecraft] (minecraft.jar) 
> 	UCH	mcp{9.19} [Minecraft Coder Pack] (minecraft.jar) 
> 	UCH	FML{8.0.99.99} [Forge Mod Loader] (forge-1.12-14.21.1.2443.jar) 
> 	UCH	forge{14.21.1.2443} [Minecraft Forge] (forge-1.12-14.21.1.2443.jar) 
> 	UCH	jecharacters{1.12.0-2.0.1} [Just Enough Characters] (JustEnoughCharacters-1.12.0-2.0.1.jar) 
> 	UCH	applecore{2.2.2} [AppleCore] (AppleCore-mc1.12-2.2.2.jar) 
> 	UCH	jei{4.7.8.95} [Just Enough Items] (jei_1.12.1-4.7.8.95.jar) 
> 	UCH	appleskin{1.0.7} [AppleSkin] (AppleSkin-mc1.12-1.0.7.jar) 
> 	**_UCE	appliedenergistics2{rv5-alpha-5} [Applied Energistics 2] (appliedenergistics2-rv5-alpha-5.jar)_** 
> 	UCH	codechickenlib{3.1.2.295} [CodeChicken Lib] (CodeChickenLib-1.12-3.1.2.295-universal.jar) 
> 	UCH	chickenchunks{2.4.0.70} [ChickenChunks] (ChickenChunks-1.12-2.4.0.70-universal.jar) 
> 	UCH	redstoneflux{2.0.1} [Redstone Flux] (RedstoneFlux-1.12-2.0.1.2-universal.jar) 
> 	UCH	cofhcore{4.3.5} [CoFH Core] (CoFHCore-1.12-4.3.5.13-universal.jar) 
> 	**_UCH	cofhworld{1.0.1} [CoFH World] (CoFHWorld-1.12-1.0.1.8-universal.jar)_** 
> 	UCH	enderstorage{1.0} [EnderStorage] (EnderStorage-1.12-2.4.0.119-universal.jar) 
> 	UCH	ic2{2.8.17-ex112} [IndustrialCraft 2] (industrialcraft-2-2.8.17-ex112.jar) 
> 	UCH	forestry{5.5.5.193} [Forestry] (forestry_1.12.1-5.5.5.193.jar) 
> 	UCH	hud{1.3.9} [Better HUD] (hud-1.3.9-1.12.jar) 
> 	UCH	inventorytweaks{1.63+dev.98.684f99a} [Inventory Tweaks] (InventoryTweaks-1.63+dev.98.jar) 
> 	UCH	journeymap{1.12-5.4.9} [JourneyMap] (journeymap-1.12-5.4.9.jar) 
> 	UCH	thermalfoundation{2.3.5} [Thermal Foundation] (ThermalFoundation-1.12-2.3.5.14-universal.jar) 
> 	UCH	thermalexpansion{5.3.5} [Thermal Expansion] (ThermalExpansion-1.12-5.3.5.19-universal.jar) 
> 	UCH	redstonearsenal{2.3.5} [Redstone Arsenal] (RedstoneArsenal-1.12-2.3.5.13-universal.jar) 
> 	UCH	rftools{6.12} [RFTools] (rftools-1.12-6.12.jar) 
> 	UCH	rftoolsdim{5.04} [RFTools Dimensions] (rftoolsdim-1.12-5.04.jar) 
> 	UCH	theoneprobe{1.4.15} [The One Probe] (theoneprobe-1.12-1.4.15.jar) 
> 	UCH	thermalcultivation{0.1.1} [Thermal Cultivation] (ThermalCultivation-1.12-0.1.1.6-universal.jar) 
> 	UCH	thermaldynamics{2.3.5} [Thermal Dynamics] (ThermalDynamics-1.12-2.3.5.12-universal.jar) 
> 	UCH	bspkrscore{at_0.9} [bspkrsCore] (Treecapitator-1.12-at_0.9a.jar) 
> 	UCH	treecapitator{at_0.9} [Treecapitator] (Treecapitator-1.12-at_0.9a.jar) 
> 	UCH	veinminer{0.38.1} [Vein Miner] (VeinMiner-1.12-0.38.1.639+134fb1e.jar) 
> 	UCH	veinminermodsupport{0.38.1} [Mod Support] (VeinMiner-1.12-0.38.1.639+134fb1e.jar) 
> 	UCH	xpbox{1.0.1.12} [XPBox] (XPBox1.12-1.0.1.12.jar) 
> 	Loaded coremods (and transformers): 
> JechCore (JustEnoughCharacters-1.12.0-2.0.1.jar)
>   me.towdium.jecharacters.core.ClassTransformer
> Inventory Tweaks Coremod (InventoryTweaks-1.63+dev.98.jar)
>   invtweaks.forge.asm.ContainerTransformer
> AppleCore (AppleCore-mc1.12-2.2.2.jar)
>   squeek.applecore.asm.TransformerModuleHandler
> 	GL info: ' Vendor: 'Intel' Version: '4.3.0 - Build 20.19.15.4549' Renderer: 'Intel(R) HD Graphics 4600'
> 	AE2 Version: alpha rv5-alpha-5 for Forge 14.22.1.2478
> 	Launched Version: 1.12-forge1.12-14.21.1.2443
> 	LWJGL: 2.9.4
> 	OpenGL: Intel(R) HD Graphics 4600 GL version 4.3.0 - Build 20.19.15.4549, Intel
> 	GL Caps: Using GL 1.3 multitexturing.
> Using GL 1.3 texture combiners.
> Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
> Shaders are available because OpenGL 2.1 is supported.
> VBOs are available because OpenGL 1.5 is supported.
> ...

Maybe because of some mod changed the items regist(Maybe COFHworld, I am not sure).
So it can't find this item.
Try fix it!

**Comment by IcedReaper** [comment]
Some further Information:
This morning I only updated AE2 to said version and CodeChickenLib and get the same crash.
After going back to version Alpha 4 (CodeChickenLib wasn't downgraded) is works again.

I also removed the configs as I thought it could be a config error as I changed some recipes.

Even in a fresh new instance with only AE2 rv5 (MC 1.12.2 Forge 14.23.0.2494) I get the same crash report.
